yarn.lock

```diff
@@ -6115,18 +5855,22 @@ react-dom@~16.3.2:
     prop-types "^15.6.0"
 
 react-dotdotdot@^1.1.0:
-  version "1.2.3"
-  resolved "https://registry.yarnpkg.com/react-dotdotdot/-/react-dotdotdot-1.2.3.tgz#45bb264153cee73471f55c51c8d1542f4880a34f"
+  version "1.3.0"
```

Comment:
We should stick to version `1.2.3`, because in `1.3.0` they change component API `useNativeClamp: false`

Reply:
Pinned down to `react-dotdotdot` to version `1.2.3`.